example.js

```diff
@@ -1,12 +1,12 @@
 'use strict'
 
 // Start a disposable node, and get access to the api
-// print the node id, and kill the temporary daemon
+// print the node id
 
 // IPFS_PATH will point to /tmp/ipfs_***** and will be
 // cleaned up when the process exits.
 
-const ipfsd = require('ipfsd-ctl')
+const ipfsd = require('.')
```

Comment:
Yes this works, though in the Node.js docs `require('./')` with a slash is commonly used. As it is pointing to the root of this module Node.js will lookup the main field in package.json and so it finds `src/index.js`. 
https://nodejs.org/api/modules.html#modules_all_together

If the examples point to the root of the module, the src files and main field in package.json could be refactored and moved without changing the examples. But I'm happy to change the examples/* to point to `../src` if you want.